.github/workflows/ci.yml

```diff
@@ -19,22 +19,28 @@ jobs:
       - uses: actions/checkout@master
 
       - name: PHP-CS-Fixer
-        uses: docker://oskarstark/php-cs-fixer-ga:2.19.0
+        uses: docker://oskarstark/php-cs-fixer-ga:3.2.1
         with:
-          args: --diff-format=udiff --dry-run --show-progress=dots --using-cache=no --verbose .
+          args: --format=txt --diff --dry-run --using-cache=no --verbose .
 
   phpstan:
     name: PHPStan
 
     runs-on: ubuntu-latest
 
+    strategy:
+      fail-fast: false
+      matrix:
+        php-version:
+          - "7.4"
```

Comment:
Should we run this on 8.1 as well?

Reply:
Yes, I have that in a forthcoming PR. There are some warnings I need to address first.